server/utils/dateUtils.ts

```diff
@@ -77,7 +77,7 @@ export class DateFormats {
    * @returns the date in the to be shown in the UI: "Thursday, 20 December 2012".
    */
   static isoDateTimeToUIDateTime(isoDate: string) {
-    return format(DateFormats.isoToDateObj(isoDate), "d MMMM y 'at' HH:mmaaa")
+    return format(DateFormats.isoToDateObj(isoDate), "d MMMM y 'at' h:mmaaa")
```

Comment:
Is this deliberate? 
This would be 12hr clock, but there is no AM/PM indicator in the format

Reply:
See the before/after screenshots in the PR descriptions